common/djangoapps/third_party_auth/urls.py

```diff
@@ -3,14 +3,20 @@
 
 from django.conf.urls import include, url
 
+from rest_framework import routers
+
 from .views import (
     IdPRedirectView,
     inactive_user_view,
     lti_login_and_complete_view,
     post_to_custom_auth_form,
-    saml_metadata_view
+    saml_metadata_view,
+    SAMLProviderConfigViewSet,
 )
 
+samlprovider_router = routers.SimpleRouter()
+samlprovider_router.register(r'^auth/samlproviderconfig', SAMLProviderConfigViewSet, basename='samlproviderconfig')
```

Comment:
I agree with @jlajoie about versioning the URLs. We should talk about maybe versioning all of them before release. I've found having non-versioned urls to be a total drag down the road.

Reply:
I added versionning with v0 so now endpoints will be like this 
```
/auth/saml/
   /v0/
      providerconfig/
      providerdata/
```